app/Main.hs

```diff
@@ -5,50 +5,48 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE RecordWildCards #-}
 module Main where
 
-import Web.Spock
-import Web.Spock.Config
-import Data.Yaml.Config (loadYamlSettings, useEnv)
-import Data.Aeson (parseJSON, FromJSON, withObject, (.:))
-import Database.Persist.TH (persistLowerCase, share, mkPersist, sqlSettings, mkMigrate)
-import Database.Persist.MySQL ( MySQLConf
-                              , createMySQLPool
-                              , SqlBackend
-                              , myConnInfo
-                              , myPoolSize
-                              , runSqlPool
-                              , printMigration
-                              )
-import Control.Monad.Logger (runStdoutLoggingT)
+import Data.Aeson ((.:))
 import Data.Text (Text)
+import Web.Spock (SpockM)
+import qualified Web.Spock as Spock
+import qualified Web.Spock.Config as Config
+import qualified Data.Yaml.Config as Yaml
+import qualified Data.Aeson as Json
```

Comment:
if we want to make the names of the used libraries obvious, shouldn't it say `as Aeson` ? 🤔